README.md

```diff
@@ -936,6 +936,8 @@ To run the test, simply invoke your favorite test runner, or execute a test file
     python test/test_download.py
     nosetests
 
+To see how to run only specific test cases, see [below](#adding-support-for-a-new-site).
```

Comment:
1. Link mentioned does not explain how to run specific test cases rather it explains how to run extractor test cases. Text should be reworded appropriately.
2. Instead of `below`, `new extractor tutorial` or so should be used with exact list's item number (if it's not possible to create a direct link).